doc/customize.doc

```diff
@@ -380,14 +380,20 @@ its name prefixed with a dollar sign, e.g.
 This was mainly added for backward compatibility.
 Note that the \c visible attribute is just a hint for doxygen.
 If no relevant information is available for a certain piece it is
-omitted even if it is set to \c yes (i.e. no empty sections are generated).
+omitted even if it is set to \c yes (i.e. no empty sections are generated).<br>
+Note also that not all elements have a \c visible attribute shown in the layout file.
+This attribute can be used to customize the output anyway but should be used used with
+caution as unexpected results (e.g.incorrect links) might appear.
```

Comment:
If I read this as a user, then what I am I supposed to do with this information? I would say that we either offer a feature that always works (and if it does not despite our best efforts it can be regarded as a bug that needs to be fixed) or we don't offer it. If we really need to add this disclaimer, I prefer not to provide it.

Reply:
I don't like disclaimers either and I agree that in case it is not working we have to see this as a bug.

I added the disclaimer as in the original version (see the 1.9.3 manual) there was already the disclaimer:
> Warning
>      at the moment you should not remove elements from the layout file as a way to hide information. Doing so can
cause broken links in the generated output!

(and as removing the line has the same effect as `visible="no"` I added the disclaimer.

I'll remove the disclaimer and reformulate about the `visible=` in case it is not shown in the layout file (or should we add everywhere in the layout file the `visible="yes"` to make it more explicit?).

Reply:
I would keep it implicit for the cases where we do not expect `visible="no"` is likely to be used, so probably for the cases where it is now not present. Just not to clutter the file too much.